src/components/Voting/Referendum/Items/ProposalItem/index.js

```diff
@@ -37,14 +69,28 @@ function BoundedCall({ call }) {
 }
 
 function ProposalItem({
-  name, createdBy, currentEndorsement, externalLink, description, userDidEndorse, boundedCall, buttonEndorseCallback, proposalIndex,
+  name,
+  createdBy,
+  currentEndorsement,
+  externalLink,
+  description,
+  userDidEndorse,
+  boundedCall,
+  buttonEndorseCallback,
+  proposalIndex,
 }) {
   return (
     <Card
       title={name}
       className={styles.cardProposalsSection}
     >
       <div>
+        <div className={styles.rowEnd}>
+          {blacklistButton(
```

Comment:
This should be `<BlacklistButton hash={hash} />` or we risk running into hook call order issues.